README.de.md

```diff
@@ -0,0 +1,97 @@
+| Du sind nicht sehr gut für die Umwelt, da Du nach nur einem Schuljahr weggeworfen werden               | Dlool ist digital, es wird kein Papier benötigt.                                                                                                                                                                                                                                                       |
+| Du sind nicht sehr praktisch                                                                            | Dlool ist auf allen Geräten verfügbar.                                                                                                                                                                                                                                                                 |
+| Einträge sind nicht sehr strukturiert                                                                    | Dlool hat ein strukturiertes System zum Eintragen von Hausaufgaben.                                                                                                                                                                                                                                    |
+| Wenn Du Ihr Hausaufgabenheft verlieren, sind alle Ihre Daten für immer verloren                         | Dlool ist digital, Du können es nicht verlieren.                                                                                                                                                                                                                                                      |
+| Jedes Jahr braucht man ein neues Hausaufgabenheft                                                        | Dlool kann für mehrere Jahre verwendet werden.                                                                                                                                                                                                                                                         |
+| Du sind an eine bestimmte Marke gebunden                                                                | Dlool ist kostenlos und quelloffen. *(Mit Dlool können Du Ihre Daten in ical oder todo.txt exportieren. Außerdem gibt es [eine Möglichkeit](https://github.com/Dlurak/dlool-scriptable/blob/main/reminders/README.md), Dlool mit Apple Reminders zu synchroniDuren.)* **(noch nicht implementiert)** |
+| Hausaufgaben, Tests und Notizen sind nicht getrennt                                                      | Dlool hat ein anderes System für Hausaufgaben, Tests und Notizen. Alles ist getrennt organiDurt, aber dennoch leicht an einem Ort zu finden.                                                                                                                                                          |
```

Comment:
This is also a great example why search and replace may seem like a good idea for many things but actually isn't very often. "organiDurt"